src/Makefile

```diff
@@ -177,19 +177,35 @@ C_CHECK_TARGETS=\
 EXPAT_OBJDIR =	node-expat/build/Release
 EXPAT.NODE =	$(EXPAT_OBJDIR)/node-expat.node
 
-$(EXPAT.NODE) :	WARN_FLAGS +=	-Wno-unused
+$(EXPAT.NODE) :	NODE_WARN_FLAGS += -Wno-unused
 $(EXPAT.NODE) :	CPPFLAGS +=	$(NODE_CPPFLAGS)
 $(EXPAT.NODE) :	CXXFLAGS +=	$(NODE_CXXFLAGS)
 $(EXPAT.NODE) :	LDFLAGS +=	$(NODE_LDFLAGS)
 $(EXPAT.NODE) :	LIBS +=		-lexpat
 
+#
+# The libuuid Node add-on follows the same pattern as EXPAT.NODE.
+#
+UUID_OBJDIR =	node-libuuid/build/Release
+UUID_SRCDIR =	node-libuuid/src
+UUID.NODE =	$(UUID_OBJDIR)/uuid.node
+
+$(UUID.NODE) :	NODE_WARN_FLAGS += -Wno-unused -Wno-write-strings
+$(UUID.NODE) :  NODE_WARN_FLAGS += -Wno-sign-compare
```

Comment:
Nit: seems misaligned with the surrounding definitions (I only mention it since you fixed the EXPAT one above).

Reply:
done